UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs

```diff
@@ -298,6 +294,36 @@ public void CmdSendBlockShuttleRecall(string adminId, string adminToken, bool to
 
 		#endregion
 
+		#region PlayerCommands
+
+		/// <summary>
+		/// Smites the selected user, gibbing him instantly.
+		/// </summary>
+		/// <param name="adminId">Id of the admin performing the action</param>
+		/// <param name="adminToken">Token that proves the admin privileges</param>
+		/// <param name="userToSmite">User Id of the user to smite</param>
+		[Server]
+		public void CmdSmitePlayer(string adminId, string adminToken, string userToSmite)
+		{
+			GameObject admin = PlayerList.Instance.GetAdmin(adminId, adminToken);
+
+			if (admin == null)
+				return;
+
+			var players = PlayerList.Instance.GetAllByUserID(userToSmite);
+			if (players.Count != 0)
+			{
+				foreach (ConnectedPlayer player in players)
+				{
+					var message = $"{player.Name} has been smiten by {PlayerList.Instance.GetByUserID(adminId).Username}: Username: {player.Username}";
+					Logger.Log(message);
+
```

Comment:
Add this here too

`UIManager.Instance.adminChatWindows.adminToAdminChat.ServerAddChatRecord(msg, null);
DiscordWebhookMessage.Instance.AddWebHookMessageToQueue(DiscordWebhookURLs.DiscordWebhookAdminLogURL, msg, "");`

Reply:
And could you reformat the message to like this one:

$"{PlayerList.Instance.GetByUserID(adminId).Username}: {state} shuttle recalling.";

Admin username, then action

Reply:
`var message = $"{PlayerList.Instance.GetByUserID(adminId).Username}: Smitted Username: {player.Username}, {player.Name}";`

Something like that maybe, though whether its smited or smitted, stupid words :p